### PR TITLE
71975 Add RUM for 1010cg

### PR DESCRIPTION
**Pull request by jadon1979**
## Summary

Add Real User Monitoring for Datadog to 1010CG 

## Related issue(s)

https://app.zenhub.com/workspaces/10-10-health-apps-5fff0cfd1462b6000e320fc7/issues/gh/department-of-veterans-affairs/va.gov-team/71975

## Testing Done

RUM doesn't officially work until it receives its first user session.

### CODE REVIEW

File: src/applications/hca/containers/App.jsx [hunk 1]
```diff
@@ -29,7 +29,7 @@ const App = props => {
     user,
   } = props;
 
-  const { isFacilitiesApiEnabled = false, isSigiEnabled = false } = features;
+  const { isFacilitiesApiEnabled = true, isSigiEnabled = false } = features;
```
Comment (longmd) [inline src/applications/hca/containers/App.jsx:32]: Please revert these to `false`.

File: src/applications/hca/containers/App.jsx [hunk 2]
```diff
@@ -57,7 +57,7 @@ const App = props => {
       const defaultViewFields = {
         'view:isLoggedIn': isLoggedIn,
         'view:isSigiEnabled': isSigiEnabled,
-        'view:isFacilitiesApiEnabled': isFacilitiesApiEnabled,
+        'view:isFacilitiesApiEnabled': true,
```
Comment (longmd) [inline src/applications/hca/containers/App.jsx:60]: Same as above. This should not be defaulted to `true`.

**Review by longmd — CHANGES_REQUESTED** [review]

File: src/applications/caregivers/utils/selectors/feature-toggles.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
+import { createSelector } from 'reselect';
+import { toggleValues } from '@department-of-veterans-affairs/platform-site-wide/selectors';
+import FEATURE_FLAG_NAMES from '@department-of-veterans-affairs/platform-utilities/featureFlagNames';
+
+const selectFeatureToggles = createSelector(
+  state => ({
+    isLoadingFeatureFlags: state?.featureToggles?.loading,
+    isBrowserMonitoringEnabled: toggleValues(state)[
+      FEATURE_FLAG_NAMES.hcaBrowserMonitoringEnabled
```
Comment (longmd) [inline src/applications/caregivers/utils/selectors/feature-toggles.js:9]: Lets go ahead and decouple this from the EZ & EZR by creating its own feature flag.

**Review by longmd — CHANGES_REQUESTED** [review]

File: src/applications/caregivers/utils/selectors/feature-toggles.js (hunk 3 above)
Comment (jadon1979) [reply src/applications/caregivers/utils/selectors/feature-toggles.js]: https://github.com/department-of-veterans-affairs/vets-api/pull/15459

File: src/platform/utilities/feature-toggles/featureFlagNames.js [hunk 4]
```diff
@@ -6,6 +6,7 @@ export default Object.freeze({
   bcasLettersUseLighthouse: 'bcas_letters_use_lighthouse',
   benefitsDocumentsUseLighthouse: 'benefits_documents_use_lighthouse',
   burialFormEnabled: 'burial_form_enabled',
+  caregiverBrowserMonitoringEnabled: 'caregiver_real_time_monitoring',
```
Comment (longmd) [inline src/platform/utilities/feature-toggles/featureFlagNames.js:9]: We want these to be a 1:1. Use `caregiver_browser_monitoring_enabled`

**Review by longmd — CHANGES_REQUESTED** [review]

**Review by longmd — APPROVED** [review]

**Review by IGallupSoCo — APPROVED** [review]
LGTM

**Pull request merged**